lib/hooks/usePrizePoolType.js

```diff
@@ -0,0 +1,92 @@
+import { useContext, useEffect } from 'react'
+import { useAtom } from 'jotai'
+import { ethers } from 'ethers'
+
+import ProxyFactory from '@pooltogether/pooltogether-contracts/abis/ProxyFactory'
+
+import { networkAtom, poolAddressesAtom, prizePoolTypeAtom } from 'lib/components/PoolUI'
+import { BLOCK_NUMBERS, CONTRACT_ADDRESSES, PrizePoolType } from 'lib/constants'
```

Comment:
Let's use PRIZE_POOL_TYPE if it's a constant coming from the constants file. It makes it easier to recognize. If I saw this as it is I might think it's a React component.